CONTRIBUTING.md

```diff
@@ -2,27 +2,23 @@
 
 ## Filing issues
 
-File issues using the standard Github issue tracker for the repo.
+File issues using the standard [Github issue tracker](https://github.com/minishift/minishift/issues) for the repository.
+Before you submit a new issue, we recommend that you:
 
-## How to become a contributor and submit your own code
+- Read the latest [release notes](https://github.com/minishift/minishift/releases) to check if the issue is known.
+- Search the list of issues to see if anyone already submitted a similar issue.
 
-### Contributor License Agreements
+## Contributing patches
 
-We'd love to accept your patches! Before we can take them, we have to jump a couple of legal hurdles.
+Thank you for your contributions! Please follow this process to submit a patch:
 
-Please fill out either the individual or corporate Contributor License Agreement (CLA).
+1. Create an issue describing your proposed change to the repository.
+1. The repository owners will triage and respond to your issue promptly.
```

Comment:
-1 for the repeated index. We should use "Numbered Lists" formatting for markdown [1] . So that the document communicates the same for command line editors , notepads and github .
[1] https://en.support.wordpress.com/markdown-quick-reference/